static/app/views/issueList/actions/utils.tsx

```diff
@@ -161,3 +162,16 @@ export function performanceIssuesSupportsIgnoreAction(
 ) {
   return !(statusDetails.ignoreWindow || statusDetails.ignoreUserWindow);
 }
+
+// A mapping of which screen sizes will trigger the column to disappear
+// e.g. 'Trend': screen.small => 'Trend' column will disappear on screen.small widths
+export const COLUMN_BREAKPOINTS = {
+  ISSUE: undefined, // Issue column is always visible
+  TREND: commonTheme.breakpoints.small,
+  AGE: commonTheme.breakpoints.xlarge,
+  SEEN: commonTheme.breakpoints.xlarge,
+  EVENTS: commonTheme.breakpoints.medium,
+  USERS: commonTheme.breakpoints.medium,
+  PRIORITY: commonTheme.breakpoints.large,
+  ASSIGNEE: commonTheme.breakpoints.xsmall,
+};
```

Comment:
I know this adds a bunch of dynamic emotion styles, but it is already a massive pain in the ass to synchronize the margins/spacing between both the header and the rows, we don't need to add column breakpoints to that list, so having this single source of truth is much cleaner.

Reply:
I like this a lot better!